src/main/java/io/cdap/plugin/gcp/gcs/actions/GCSBucketDelete.java

```diff
@@ -119,13 +128,80 @@ public void run(ActionContext context) throws Exception {
       if (fs.exists(gcsPath)) {
         try {
           fs.delete(gcsPath, true);
+          deleteCount++;
         } catch (IOException e) {
           LOG.warn(
             String.format("Failed to delete path '%s'", gcsPath)
           );
         }
       }
     }
+
+    for (GCSPath gcsPath : gcsPathsWild) {
+      String regex = getRegex(gcsPath);
+      try {
+        fs = new Path(gcsPath.getUri()).getFileSystem(configuration);
+      } catch (IOException e) {
+        LOG.info("Failed deleting file " + gcsPath.getUri().getPath() + ", " + e.getMessage());
+        // no-op.
+        continue;
+      }
+      Page<Blob> blobs = storage.list(gcsPathsWild.get(0).getBucket());
```

Comment:
`gcsPathsWild.get(0)` -> gcsPath

Reply:
test when user input `gs://bucket*` what will happen here to `getBucket()`, try to handle this error case

Reply:
Might received error when use `gs://bucket*`. Try validate and received message: 

Invalid bucket name in path 'test_664_*'. Bucket name should only contain lower case alphanumeric, '-'. '_' and '.'. Please follow GCS naming convention: https://cloud.google.com/storage/docs/naming-buckets